src/plugins/api/qqmlsettings.cpp

```diff
@@ -343,8 +343,13 @@ void QQmlSettingsPrivate::load()
         const QVariant currentValue = instance()->value(property.name(), previousValue);
 
         if (!currentValue.isNull() && (!previousValue.isValid()
-                                       || (currentValue.canConvert(previousValue.userType()) && previousValue != currentValue))) {
-            property.write(q, currentValue);
+#if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
```

Comment:
Please use define `MU_QT5_COMPAT` (if defined, then Qt5)
It will be easier for us to find all these places and clean them when we completely switch to Qt6 and remove support for Qt5

Reply:
Done.
What about the remaining one in https://github.com/musescore/MuseScore/blob/86f52447126b1b61b55adcfd414bb621466e1de5/thirdparty/KDDockWidgets/src/Qt5Qt6Compat_p.h#L21

Reply:
Lemmi fix that too...

Reply:
I believed that chage to have been from us, not them?

Reply:
These are their changes, not ours.
We were also surprised that they made compatibility for both Qt5 and Qt6 a long time ago.

Reply:
OK, so just ignore my commit 3 (and 2)?

Reply:
I think yes, it will be better this way

Reply:
Do you want me to remove tham or are you going to jut cherry-pick the 1st commit?

Reply:
It's better if you remove it so I can just click on the merge button :)

Reply:
Done ;-)